webapp/channels/src/components/announcement_bar/default_announcement_bar/announcement_bar.tsx

```diff
@@ -185,6 +186,7 @@ export default class AnnouncementBar extends React.PureComponent<Props, State> {
                 <button
                     onClick={this.props.onButtonClick}
                     disabled={this.props.ctaDisabled}
+                    className='btn btn-tertiary btn-xs btn-inverted'
```

Comment:
Is this intended to be smaller font than the text? Just double checking that this is how the action buttons look in the announcement banner: 
<img width="1296" alt="Screenshot 2025-05-28 at 4 43 42 PM" src="https://github.com/user-attachments/assets/14d596d0-fe7d-435e-8e26-1d996c5e0317" />
<img width="1094" alt="Screenshot 2025-05-28 at 4 42 17 PM" src="https://github.com/user-attachments/assets/54d62ed2-0dbd-45e8-a5ad-57a3d9686384" />